components/brave_rewards/browser/database/publisher_info_database.cc

```diff
@@ -1023,10 +1024,6 @@ void PublisherInfoDatabase::GetPendingContributions(
   bool initialized = Init();
   DCHECK(initialized);
 
-  if (!initialized) {
```

Comment:
why was this removed?

Reply:
Don't know! Seems like an accident.

Reply:
Basically, @bridiver @NejcZdovc we shouldn't put `DCHECK(initialized)` everywhere, because it is a normal situation - there is a notable percentage of broken databases (for various reasons, e.g. filesystem failures). Rather we should gracefully shutdown the service if the DB was not initialized properly. Or even try to restore it using SQLite tools